bears/general/LineContinuationBear.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.bearlib.languages.Language import Language
+
+
+class LineContinuationBear(LocalBear):
+    def run(self,
+            filename,
+            file,
+            language,
+            ):
+        try:
+            lang = Language[language]
+            line_continuation = lang.get_default_version().line_continuation
+        except Exception:
+            logging.error('Language {} is not yet supported.'.format(language))
+            return
+
+        for line_number, line in enumerate(file):
+            if len(line) - 1:
```

Comment:
What if `len(line) == 0`?
Maybe that would never happen - but it's better to be explicit IMO, e.g. `len(line) > 1`

Reply:
Actually given my comment below,  `len(line) > whatever` here is a bad condition.

Reply:
Yeah, any other options to solve it?